react/week1/todo/src/toDoList.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+function Checkbox() {
+    return (
+        <div>
+            <input type="checkbox" />
+        </div>
+    )
+}
+
+function Deadline(props) {
+    const deadline = new Date(props.date.year, props.date.month - 1, props.date.day);
+    const today = new Date();
+    const status = deadline > today ? "valid" : "expired";
+    return (
+        <div>
+            <p>Deadline date: {props.date.day}/ {props.date.month}/ {props.date.year}</p>
+            <p>Status : {status}</p>
+        </div>
+    );
+}
+
+function ToDoItems(props) {
+    return (
+        <li>
```

Comment:
See if you can use a library to handle dates instead. for instance moment.js

Reply:
sure, will try to do that. Thanks for your time and feedback

Reply:
It's done, thank you so much